### DISCUSSION TITLE
Version for OBS 27.2.4

### DISCUSSION THREAD

**Discussion by billborez** [WarmUpTill/SceneSwitcher#574] [Q&A] [answered]
I've had to downgrade OBS to 27.2.4 to get NDI input to work but I'm not seeing advanced sceneswitcher plugin after installation today, is there a specific version I need to get it to run in OBS 27?

**Comment by WarmUpTill** [comment, marked as answer]
Yes, you will have to drop back down to version 1.17.x
https://github.com/WarmUpTill/SceneSwitcher/releases/tag/1.17.7

**Reply by billborez** [reply]
Perfect, that's working great now thanks.
